validitymap.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package amppackager
+
+import (
+	"net/http"
+)
+
+type ValidityMap struct {
+	validityMap []byte
+}
+
+func NewValidityMap() (*ValidityMap, error) {
+	this := new(ValidityMap)
+	this.validityMap = []byte("\xA0")
+	return this, nil
+}
+
+func (this ValidityMap) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
+	println("path", req.URL.Path)
+	if req.URL.Path == ValidityMapURL {
+		resp.Header().Set("Accept-Ranges", "bytes")
```

Comment:
You can use https://golang.org/pkg/net/http/#ServeContent which should "do the right thing" w.r.t. range requests. (Though you'll still need to specify content-type and cache-control separately.)

Reply:
Done